applications/nrf_desktop/description.rst

```diff
@@ -952,6 +952,24 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for information abou
 
 .. nrf_desktop_fastpair_important_end
 
+nRF54L MCUboot provisioning
```

Comment:
Can we avoid target names in section titles? You can find a few suggestions below:

```
MCUboot provisioning using the key storage peripheral
```
or
```
MCUboot provisioning using the KMU peripheral
```
or
```
MCUboot provisioning using the HW storage
```

The KMU is also available for nRF53 Series.

And then we can list supported targets or even provide a reference to the list that is maintained in the general MCUboot documentation

Reply:
I intentionally prefer to keep `nRF54L` in the section name as currently it's the only SoC Series that supports the feature. Even Kconfigs and build system have SoC Series dependencies. I am worried that making the documentation more generic could lead to false impression that other SoC Series might also support these features.

Reply:
Also, using `nRF54L` in the title allows to skip the section for other SoC Series users without reading.